proto/p4/config/p4info.proto

```diff
@@ -136,8 +138,22 @@ message ActionProfile {
 
 message Counter {
   Preamble preamble = 1;
-  // 0 (default value) if the counter is not direct, the id of the table to
-  // which the counter is attached otherwise
+  // in the current P4 spec the attribute is called 'type', I believe 'unit' is
+  // more appropriate for this and I hope to see the spec updated
```

Comment:
This comment seems more like code review dialog than information to assist the developer.